snapcraft.yaml

```diff
@@ -18,7 +18,8 @@ parts:
       - wget
     build-environment:
       - RELEASE: "24.04"
-      - BASE: ubuntu-base-${RELEASE}-base-${CRAFT_ARCH_BUILD_FOR}.tar.gz
+      - DOT_RELASE: "1"
+      - BASE: ubuntu-base-${RELEASE}.${DOT_RELASE}-base-${CRAFT_ARCH_BUILD_FOR}.tar.gz
```

Comment:
Maybe add this using `BASE: ubuntu-base-${RELEASE}${DOT_RELEASE:+.$DOT_RELEASE}-base-${CRAFT_ARCH_BUILD_FOR}.tar.gz` ? That way, if DOT_RELEASE isn't defined, it will use only the RELEASE version, and

Reply:
Good point, I've changed now (actually to `${DOT_RELEASE+.$DOT_RELEASE}`, I don't think you need the colon). I was not sure if snapcraft would process this as expected but seems to be the case.

Reply:
I've rethought about the colon and put back.

Reply:
I tested it and yes, the colon is needed.